Fix the build system to use MPM_SUBDIR_NAME (which points to the directory
with the MPM in) instead of MPM_NAME (which returns the name of the MPM);
they differ since the MPM is in experimental/.

Index: apache2.2/server/mpm/experimental/itk/config.m4
===================================================================
--- apache2.2.orig/server/mpm/experimental/itk/config.m4	2007-01-29 21:03:51.000000000 +0100
+++ apache2.2/server/mpm/experimental/itk/config.m4	2007-01-29 21:03:57.000000000 +0100
@@ -1,3 +1,3 @@
 if test "$MPM_NAME" = "itk" ; then
-    APACHE_FAST_OUTPUT(server/mpm/$MPM_NAME/Makefile)
+    APACHE_FAST_OUTPUT(server/mpm/$MPM_SUBDIR_NAME/Makefile)
 fi
